packages/web/hooks/use-export.ts

```
@@ -0,0 +1,455 @@
+
+  /**
+   * Share export result
+   */
+  const shareResult = useCallback(
+    async (result: ExportResult) => {
+      try {
+        // Sharing not implemented in current export service
+        throw new Error('Sharing is not yet supported')
+        if (showToasts) {
+          toast.success('Export shared successfully')
+        }
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)